src/librustdoc/json/conversions.rs

```diff
@@ -4,14 +4,14 @@
 
 #![allow(rustc::default_hash_types)]
 
+use rustc_abi::ExternAbi as RustcAbi;
```

Comment:
wasn't entirely sure whether you would prefer to preserve this rename or not.

Reply:
Doesn't really make sense anymore. Do you mind removing this renaming please?

Reply:
Sure!